{
	"resourceType": "Condition",
	"id": "e3877bd7-9a91-bbab-e54e-3b9207b6c1e1",
	"meta": {
		"profile": [
			"http://hl7.org/fhir/us/core/StructureDefinition/us-core-condition-encounter-diagnosis"
		]
	},
	"clinicalStatus": {
		"coding": [
			{
				"system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
				"code": "resolved"
			}
		]
	},
	"verificationStatus": {
		"coding": [
			{
				"system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
				"code": "confirmed"
			}
		]
	},
	"category": [
		{
			"coding": [
				{
					"system": "http://terminology.hl7.org/CodeSystem/condition-category",
					"code": "encounter-diagnosis",
					"display": "Encounter Diagnosis"
				}
			]
		}
	],
	"code": {
		"coding": [
			{
				"system": "http://snomed.info/sct",
				"code": "840539006",
				"display": "Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)"
			}
		],
		"text": "Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)"
	},
	"subject": {
		"reference": "urn:uuid:69068937-ff52-ebc2-f140-f9ee77fad999"
	},
	"encounter": {
		"reference": "urn:uuid:37dcafb5-8258-ec58-ad6d-e80db0397a90"
	},
	"onsetDateTime": "2020-11-03T17:27:39-07:00",
	"abatementDateTime": "2020-11-20T19:27:39-07:00",
	"recordedDate": "2020-11-03T17:27:39-07:00"
}